lib/error-handler.js

```diff
@@ -17,6 +17,9 @@ module.exports = settings => {
       });
     }
     // Error thrown in AJAX call
-    res.json(error);
+    if (!settings.verboseErrors) {
+      return res.json({ message: 'Fetch failed' });
```

Comment:
This should go higher up because we want it to apply to all errors, in all circumstances, not just XHR fetches.

I'd overwrite the error message, and then just let it continue to the normal error handlers.

Reply:
good point sir